src/lazy/expanded/compiler.rs

```diff
@@ -94,7 +95,7 @@ impl ExpansionSingleton {
 }
 
 /// Validates a given TDL expression and compiles it into a `TemplateMacro` that can be added
-/// to a [`MacroTable`](crate::lazy::expanded::macro_table::MacroTable).
+/// to a [`MacroTable`].
```

Comment:
🗺️ This explicit path was made redundant because this module now imports `MacroTable`.